charts/apisix-ingress-controller/values.yaml

```diff
@@ -124,3 +124,9 @@ serviceMonitor:
   enabled: false
   namespace: "monitoring"
   interval: 15s
+  ## @param serviceMonitor.labels ServiceMonitor extra labels
```

Comment:
You can delete the extra # to keep it consistent with other comments.
```suggestion
  # @param serviceMonitor.labels ServiceMonitor extra labels
```

Reply:
done!